usr/share/gamescope-session/device-quirks

```diff
@@ -13,16 +13,30 @@ OXP_LIST="ONE XPLAYER:ONEXPLAYER 1 T08:ONEXPLAYER 1S A08:ONEXPLAYER 1S T08:ONEXP
 AOK_LIST="AOKZOE A1 AR07"
 if [[ ":$OXP_LIST:" =~ ":$SYS_ID:"  ]] || [[  ":$AOK_LIST:" =~ ":$SYS_ID:"   ]]; then
   # Dependent on a special --force-orientation option in gamescope
-  if ( gamescope --help 2>&1 | grep -e "--force-orientation" > /dev/null ) ; then
-    export GAMESCOPECMD="/usr/bin/gamescope \
-      -e \
-      --generate-drm-mode fixed \
-      --xwayland-count 2 \
-      -O *,eDP-1 \
-      --default-touch-mode 4 \
-      --hide-cursor-delay 3000 \
-      --fade-out-duration 200 \
-      --force-orientation left "
+  if ( gamescope --help 2>&1 | grep -e "--force-external-orientation" > /dev/null ) ; then
```

Comment:
Be careful with this option change. This might break AMD OXP if using gamescope-git. Maybe the ifs should be reversed?

Reply:
Yeah, I'll have to look into that. I'm barely awake at the moment so I'm having issues concentrating on the logic.

Reply:
Updated with a better logic. I tested on both my OXP devices and everything seems to work fine, but without being able to echo $GAMESCOPECMD i'm not 100% sure everything detected as expected. Please test.